skt/kerneltree.py

```diff
@@ -334,7 +335,7 @@ def merge_patchwork_patch(self, uri):
         status, output = self.__git_cmd_pipe(patch_content,
                                              "am", "-",
                                              cwd=self.wdir)
-        if status != 0:
+        if status != SKT_SUCCESS:
```

Comment:
Status here is the return code from subprocess call, so this one shouldn't be compared with the result codes